composer/utils/data.py

```diff
@@ -93,26 +98,60 @@ def pil_image_collate(batch: List[Tuple[Image.Image, Union[Image.Image, Tensor]]
     return image_tensor, target_tensor
 
 
-def add_dataset_transform(dataset, transform):
-    """Flexibly add a transform to the dataset's collection of transforms.
+def add_dataset_transform(dataset: VisionDataset,
+                          transform: Callable,
+                          is_tensor_transform: bool = False) -> torch.utils.data.Dataset:
+    """Add a transform to a dataset's collection of transforms. Inserts the transform
+    before or after torchvision.transforms.ToTensor(), or at the end of the collection of
+    ToTensor() is not present.
 
     Args:
-        dataset: A torchvision-like dataset
-        transform: Function to be added to the dataset's collection of transforms
+        dataset (torch.utils.data.Dataset): A torchvision-like dataset
+        transform (Callable): Function to be added to the dataset's collection of
+            transforms
+        is_tensor_transform (bool): Whether `transform` acts on data of the type
```

Comment:
RST uses double ticks for code literal blocks (I made this mistake wayy to many times, then the docs come out in italics :D) Not going to mark all, but please double check.

```suggestion
        is_tensor_transform (bool): Whether ``transform`` acts on data of the type
```